### ISSUE TITLE
Check for obsoleted(very old) Go versions on CI files

### ISSUE THREAD

**Issue by cristaloleg** [Quasilyte/repolint#1]
Check `.travis.yml`, `.circleci`(any other CI/CD service) files for the old versions (1.9 and older) and suggest to update them.

**Comment by quasilyte** [comment]
I like the idea of checking `.travis.yml` and `.circleci`.
We can also run their associated linters (if they exist) to catch more errors.

**Comment by quasilyte** [comment]
For `.travis.yml` `travis-lint` is now executed.
I don't know anything that can validate `.circleci`.

But we can do proposed Go version checks though. :)

**Comment by cristaloleg** [comment]
Can we just parse yaml and check Go version manually?

**Comment by quasilyte** [comment]
Of course, I told about another thing, linting config files for non-existing properties, for example.
Checking Go version is go-specific, this is why it's less attractive right now.

**Comment by cristaloleg** [comment]
But it can be extended till whatever you want 😎 (Go was just an example, biased one)
